.github/workflows/ruby.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+# This workflow will download a prebuilt Ruby version, install dependencies 
+# and run tests with Rake
+# For more information see: https://github.com/marketplace/actions/setup-ruby-jruby-and-truffleruby
+
+name: Ruby
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
```

Comment:
is the specific Ruby version to avoid accidentally having it break due to breaking changes in Ruby itself? Is there anything particular about `473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e` ?

Reply:
When I set up continuous integration (CI) using GitHub Actions, it automatically creates a template for ruby and it automatically added that commit. It refers to this https://github.com/ruby/setup-ruby/releases/tag/v1.61.0. I can put v1 if it doesn't look right

Reply:
This is not the ruby version, rather, it is the setup-ruby action version. The actual ruby version will be taken from here https://github.com/SUSE-Enceladus/public-cloud-info-service/blob/master/.ruby-version according to the doc.

Reply:
ah ok, thanks for clarifying... I dont know enough about it to say whether it looks right, don't block the change on this question for sure